src/react/databrowser/objects/ObjectListTable.jsx

```diff
@@ -78,6 +79,14 @@ export default function ObjectListTable({
     [dispatch],
   );
 
+  const query = useQuery();
+  const objectKey = query.get('prefix');
+  const versionId = query.get('versionId');
+
+  useEffect(() => {
+    dispatch(toggleObject(objectKey, versionId ? versionId : undefined));
```

Comment:
I see in the ObjectRow that before togglingAnObject it is dispatching `dispatch(toggleAllObjects(false));` do we nned to do so here as well ? What does `dispatch(toggleAllObjects(false));` exactly do ? I guess it is dis-selecting all the line of the table, if so I think we should likely not do that as it would loose the selection context of the user as soon as the user is checking one of the object _they_ selected

Reply:
The issue with removing the `dispatch(toggleAllObjects(false));` is that given an object is already selected, it would not be possible now to check another object, because as soon as I click on another object, its a multiple selection state :(